[Decompression] Update SQL for finding job ID

charislam opened the pull request:
The SQL query given for finding job ID was updated. There is an easier
way to do it now, without an unnecessary `JOIN`.

# Links

Fixes #1443 

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

charislam committed:
[Decompression] Update SQL for finding job ID

The SQL query given for finding job ID was updated. There is an easier
way to do it now, without an unnecessary `JOIN`.

Loquacity commented on timescaledb/how-to-guides/compression/decompress-chunks.md:
```diff
@@ -6,43 +6,57 @@ tags: [decompression]
 </highlight>
 
 These are the main steps for decompressing chunks in preparation for inserting
 or backfilling data:
+
 1.  Temporarily turn off any existing compression policy. This stops the policy
     trying to compress chunks that you are currently working on.
 1.  Decompress chunks.
 1.  Perform the insertion or backfill.
-1.  Re-enable the compression policy. This re-compresses the chunks you worked on.
+1.  Re-enable the compression policy. This re-compresses the chunks you worked
+    on.
```
Could this be refactored into a procedure?

charislam committed:
Rearrange decompress and backfilling sections

While converting backfilling instructions to a procedure, noticed a lot
of repeated sections between these two pages. Backfilling was already
documented (twice on two different pages). Removed the repetition and
cleaned up a few things to be more in line with our current style.

Loquacity commented on timescaledb/how-to-guides/compression/backfill-historical-data.md:
```diff
@@ -6,121 +6,114 @@ keywords: [compression, backfill, hypertables]
+## Backfill with a supplied function
 
-## Backfilling using the `decompress_backfill` procedure
+To simplify backfilling, you can use the [backfilling
+functions][timescaledb-extras-backfill] in the [TimescaleDB
+extras][timescaledb-extras] GitHub repository. In particular, the
+`decompress_backfill` procedure automates many of the backfilling steps for you.
```
```suggestion
To make backfilling easier, you can use the 
[backfilling functions][timescaledb-extras-backfill] in the 
[TimescaleDB extras][timescaledb-extras] GitHub repository. In particular, the
`decompress_backfill` procedure automates many of the backfilling steps for you.
```

Loquacity reviewed (approved):
Just a small one.

charislam committed:
Minor cleanup

Co-authored-by: Lana Brindley <github@lanabrindley.com>
